lib/src/build_context_impl.dart

```diff
@@ -39,12 +39,12 @@ extension MediaQueryExt on BuildContext {
 }
 
 extension NavigatorExt on BuildContext {
-  Future<T> push<T>(Route<T> route) => Navigator.push(this, route);
+  Future<T?> push<T>(Route<T> route) => Navigator.push(this, route);
 
-  void pop<T extends Object>([T result]) => Navigator.pop(this, result);
+  void pop<T extends Object>([T? result]) => Navigator.pop(this, result);
 
-  Future<Object> pushNamed<T>(String routeName, {Object arguments}) =>
-      Navigator.pushNamed(this, routeName, arguments: arguments);
+  Future<Object?> pushNamed<T>(String routeName, {Object? arguments}) =>
+      Navigator.pushNamed<T>(this, routeName, arguments: arguments);
```

Comment:
I think here one `Navigator.pushNamed` expects a nullable value so it should be: `Navigator.pushNamed<T?>`

Reply:
Yes, this makes sense. I changed it 👍